index.html

```diff
@@ -17,16 +17,16 @@ <h1>My Color Aura</h1>
         <div class="wrapper">
             <section>
                 <ul class="moodUl">
-                    <li><button class="moods" value="red">stressed</button></li>
-                    <li><button class="moods" value="pink">romantic</button></li>
-                    <li><button class="moods" value="purple">powerful</button></li>
-                    <li><button class="moods" value="navy">mysterious</button></li>
-                    <li><button class="moods" value="blue">melancholic</button></li>
-                    <li><button class="moods" value="aqua">peaceful</button></li>
-                    <li><button class="moods" value="green">healthy</button></li>
-                    <li><button class="moods" value="lime">balanced</button></li>
-                    <li><button class="moods" value="yellow">joyful</button></li>
-                    <li><button class="moods" value="orange">energetic</button></li>
+                    <li><button class="moods" value="DC143C">stressed</button></li>
+                    <li><button class="moods" value="FFC0CB">romantic</button></li>
+                    <li><button class="moods" value="663399">powerful</button></li>
+                    <li><button class="moods" value="000080">mysterious</button></li>
+                    <li><button class="moods" value="4682B4">melancholic</button></li>
+                    <li><button class="moods" value="00FFFF">peaceful</button></li>
+                    <li><button class="moods" value="228B22">healthy</button></li>
+                    <li><button class="moods" value="00FF00">balanced</button></li>
+                    <li><button class="moods" value="FFFF00">joyful</button></li>
+                    <li><button class="moods" value="FF8C00">energetic</button></li>
```

Comment:
Suggestion: now that we have more flexibility with the new API, how do you feel about adding additional buttons with a wider range of moods? Can now access any other colors we want (black, grey, etc)

Reply:
yes love the idea of expanding on our buttons; the new API gives us more flexibility.